opennms-webapp/src/main/webapp/login.jsp

```diff
@@ -108,8 +108,9 @@
     </div>
   </div>
 
-  <div class="" style="color: white; position: absolute; bottom: 0px; right: 0px; font-size: 3em; padding: 20pt 20pt 5pt 20pt">
-    <a href="https://github.com/OpenNMS/opennms.git" title="Fork us on Github"><i class="fa fa-github"></i></a>
+  <div class="" style="position: absolute; bottom: 0px; right: 0px; font-size: 3em; padding: 20pt 20pt 5pt 20pt">
+    <a href="https://docs.opennms.org" class="text-light" title="Show documentation"><i class="fa fa-book" aria-hidden="true"></i></a>
```

Comment:
Could you make it point to https://docs.opennms.org/opennms/releases/latest/index.html rather than making them browse around to the latest version?

Reply:
The reason why I pointed to docs.opennms.org is that pointing to latest will most likely not be the version the user is running. 

The following alternatives came to mind:
 - Point to the actual release. This AFAIK cannot be done
 - Point to a possible newer documentation. @RangerRick suggestion.
 - Point to the landing page. Current implementation.

I opted for the landing page. 
I am not bound to this option however.

Reply:
Good point.  You could at least point to the *opennms* landing page, I suppose: http://docs.opennms.org/opennms/

The version one should be doable as long as it's not a snapshot version.  You could just generate the link.  Still, that's potentially dangerous if we're not validating things well.

Reply:
We are using a versioned link mechanism already in Admin -> Help, maybe we can reuse something:

![screenshot 2019-01-29 at 18 51 29](https://user-images.githubusercontent.com/1095181/51928756-ec022000-23f6-11e9-82a1-6f92d2992511.png)